src/test/java/org/broadinstitute/hellbender/engine/GATKToolUnitTest.java

```diff
@@ -566,12 +621,120 @@ public void testCreateVCFWriterDefaults(
+        Assert.assertFalse(classes.isEmpty());
+        Assert.assertEquals(annots.size(),classes.size());
+        for(Class<?> found : classes) {
+            Assert.assertTrue(annots.stream().anyMatch(a -> a.getClass()==found));
         }
+    }
+
+    @Test
+    public void testExcludeAnnotation(){
+        String[] args = {"--"+StandardArgumentDefinitions.ENABLE_ALL_ANNOTATIONS, "-AX", "Coverage"};
+
+        final TestGATKToolWithVariants tool = createTestVariantTool(args);
+        Collection<Annotation> annots = tool.makeVariantAnnotations();
+
+        // Asserting that the annotation was excluded
+        Assert.assertFalse(annots.stream().anyMatch(a -> a.getClass()==Coverage.class));
+    }
+
+    @Test
+    public void testIncludeGroupAnnotations(){
+        String[] args = {"-G", StandardAnnotation.class.getSimpleName()};
+
+        final TestGATKToolWithVariants tool = createTestVariantTool(args);
+        Collection<Annotation> annots = tool.makeVariantAnnotations();
+
+        // Asserting that a standard annotation was included but not everything
+        Assert.assertTrue(annots.stream().anyMatch(a -> a.getClass()==Coverage.class));
+        Assert.assertFalse(annots.stream().anyMatch(a -> a.getClass()==StandardAnnotation.class));
+        Assert.assertFalse(annots.stream().anyMatch(a -> a.getClass()==ClippingRankSumTest.class));
+    }
+
+
+    @Test
+    public void testIncludeAnnotations(){
+        String[] args = {"-A", Coverage.class.getSimpleName()};
+
+        final TestGATKToolWithVariants tool = createTestVariantTool(args);
+        Collection<Annotation> annots = tool.makeVariantAnnotations();
+
+        // Asserting coverage was added
+        Assert.assertTrue(annots.stream().anyMatch(a -> a.getClass()==Coverage.class));
+    }
+
+    @Test
+    public void testMakeDefaultAnnotations() {
+        String[] args = null;
+
+        final TestGATKToolWithDefaultAnnotations tool = createTestVariantTool(new TestGATKToolWithDefaultAnnotations(), args);
+        Collection<Annotation> annots = tool.makeVariantAnnotations();
+
+        // Asserting coverage was added by default
+        Assert.assertTrue(annots.stream().anyMatch(a -> a.getClass()==Coverage.class));
+    }
+
+    @Test
+    public void testMakeDefaultAnnotationGroups() {
+        String[] args = null;
+
+        final TestGATKToolWithDefaultAnnotationGroups tool = createTestVariantTool(new TestGATKToolWithDefaultAnnotationGroups(), args);
+        Collection<Annotation> annots = tool.makeVariantAnnotations();
+
+        // Asserting that a standard annotation was included but not everything
+        Assert.assertTrue(annots.stream().anyMatch(a -> a.getClass()==Coverage.class));
+        Assert.assertFalse(annots.stream().anyMatch(a -> a.getClass()==StandardAnnotation.class));
+        Assert.assertFalse(annots.stream().anyMatch(a -> a.getClass()==ClippingRankSumTest.class));
+    }
+
+    @Test
+    public void testClearDefaultAnnotations() {
```

Comment:
Add a second test that clears the defaults on `TestGATKToolWithDefaultAnnotations` (to test that this works for tools that specify individual annotations instead of annotation groups)